plugins/event_notification/providers/push/lib/model/PushNotificationTemplate.php

```diff
@@ -85,9 +105,16 @@ public function getQueueKey($contentParameters, $partnerId = null, kScope $scope
         // sort array according to created keys
         ksort($contentParametersValues);
         
-        $contentParamsHash = md5($partnerId . '_' . implode( '_' , array_values($contentParametersValues) ) );
-        // prepare queue key to return
-        return 'pn_' . $templateId . '_' . $contentParamsHash;
+        $queueContentParams = $partnerId . '_' . implode( '_' , array_values($contentParametersValues));
+        if($returnRaw)
+        	return 'pn_' . $templateId . '_md5s_' . $queueContentParams . "_md5e";
```

Comment:
@tan-tan-kanarek  Changed it to use prefix & postfix similar to what we do when indexing to sphinx